### PR TITLE
Switch to yaml issue templates (forms)

### LINKED ISSUE #1541: Contributes that open an issue but don't follow the provided template
**Topic**
Please select what topic your discussion is about.
- [ ] Coding
- [x] Contribution
- [ ] Languages
- [ ] Setup
- [ ] Other. Please specify:

**Discussion**
I think that this is a real problem. Over the past few weeks i noticed allot of people opening bug reports or feature requests without following the template properly.

When the provided template isnt used its hard to understand why the contributor thinks its a good feature to implement or how to reproduce a bug or something like this.

Because this happens the devs need to ask this questions to really understand whats happening or what it is they want and why they want it.

This takes up allot of time for u guys.

In my opinion there need to be a rule or something that the template must be followed otherwise it just get closed or something.

in the contribution guidelines (readme) u cant find anything about following the template.

And in the readme of templates themself there is inconsistency.

> Also please leave all headers except this one in the issue, so that it is clear what belongs where.

I could only find this in the template of feature request. I cant find anything in the others.

Ofcourse there can be some value in the issue u opened up but it isnt that hard to describe it right?

### PR DESCRIPTION
**Pull request by ChunkyProgrammer**
---
Switch to yaml issue templates (forms)
---

**Important note**
Please note that only PrestoN is able to merge Pull Requests into master.

**Pull Request Type**
Please select what type of pull request this is:
- [ ] Bugfix
- [x] Feature Implementation

**Related issue**
closes #1541

**Description**
This PR will disable blank issues & require issues to follow the template

To preview:
https://github.com/ChunkyProgrammer/github-issue-form-test/issues/new/choose

if ^ link outdated try
https://github.com/efb4f5ff-1298-471a-8973-3d47447115dc/github-issue-form-test/issues/new/choose

**Screenshots (if appropriate)**
(red star indicated required)
Bug Report
image

Feature Request
image

Discussions (Might be removed in favour of discussions tab)
image

Questions (Might be removed in favour of discussions tab)
image

**Testing (for code that is not small enough to be easily understandable)**
Has this pull request been tested?
Please describe shortly how you tested it and whether there are any ramifications remaining. 
I tested this PR by creating issues on this repo
https://github.com/ChunkyProgrammer/github-issue-form-test

if ^ link outdated try
https://github.com/efb4f5ff-1298-471a-8973-3d47447115dc/github-issue-form-test/issues/new/choose

**Desktop (please complete the following information):**
 - OS: Windows
 - OS Version: 10
 - FreeTube version: 0.13.2

### CODE REVIEW

**Comment by ChunkyProgrammer** [comment]
I think more discussion needs to be done on whether this should be implemented or not

**Comment by PrestonN** [comment]
Looking at a glance, I do like this. I wasn't aware GitHub had something like this. This should help quite a bit with getting the information we need for submissions.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Ah i was searching for this some time ago because i saw that the element repo used this. But i forgot about it again, this would be really nice to implement.

Maybe say this "This bug persist in the latest nightly build" instead of saying "im using the latest version"

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Maybe the forms can also be used for the PR's

https://github.com/FreeTubeApp/FreeTube/issues/1512

**Comment by ChunkyProgrammer** [comment]
> Maybe the forms can also be used for the PR's
> 
> #1512

From what I can tell, PR templates don't support the yaml format & I have not seen any projects using it for PR Templates
["Pull request template filenames are not case sensitive, and can have an extension such as .md or .txt."](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates)

**Review by peepopoggers — APPROVED** [review]
Looks great, and just to make sure, I checked out your repo as well to see it in action. I would like @PrestonN 's confirmation for this but this looks all good.

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (ChunkyProgrammer) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:4]: This would require the label to be changed on the repo
(not difficult to do but it would ideally be changed shortly after this PR is merged)
image 
Click edit & add the emoji to the label

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (ChunkyProgrammer) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:13]: *encountered

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (ChunkyProgrammer) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:9]: should probably only have "description:" once

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (ChunkyProgrammer) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:11]: says "feature request" but it's a "bug report"

File: .github/ISSUE_TEMPLATE/feature_request.yaml
Comment (ChunkyProgrammer) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:4]: Same thing as bug label

**Review by ChunkyProgrammer — COMMENTED** [review]
a few suggestions & considerations  (also: fix the indents)

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:4]: oh yeah no problem we can just delete the label. i just thought it would look nice and that it would somehow know what a beelte is lol

File: .github/ISSUE_TEMPLATE/discussion.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [inline .github/ISSUE_TEMPLATE/discussion.yaml:12]: Is it possible to make at least one checkbox required?

File: .github/ISSUE_TEMPLATE/question.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [inline .github/ISSUE_TEMPLATE/question.yaml:12]: Is it possible to make at least one checkbox required?

File: .github/ISSUE_TEMPLATE/discussion.yaml
Comment (ChunkyProgrammer) [reply .github/ISSUE_TEMPLATE/discussion.yaml:12]: I don't think so (maybe use a drop down list instead?)

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/discussion.yaml:12]: How would that work? What if its applicable to multiple subjects?

Comment (ChunkyProgrammer) [reply .github/ISSUE_TEMPLATE/discussion.yaml:12]: Adding "multiple:true" to the attributes would allow multiple subjects to be selected

Ex:
```
  - type: dropdown
    id: browsers
    attributes:
      label: What browsers are you seeing the problem on?
      multiple: true
      options:
        - Firefox
        - Chrome
        - Safari
        - Microsoft Edge
```
Reference: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

File: .github/ISSUE_TEMPLATE/question.yaml
Comment (peepopoggers) [reply .github/ISSUE_TEMPLATE/question.yaml:12]: Good catch :+1:

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@ChunkyProgrammer is there an easy way to fix the indents. I was really struggling with the indentation when i wrote the code.

**Comment by ChunkyProgrammer** [comment]
> @ChunkyProgrammer is there an easy way to fix the indents. I was really struggling with the indentation when i wrote the code.

If you have visual studio code installed then I'd suggest copy the code to a file and opening it in VS Code (then formatting the document).
If you don't then I can probably fix the indents

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> 
> 
> > @ChunkyProgrammer is there an easy way to fix the indents. I was really struggling with the indentation when i wrote the code.
> 
> If you have visual studio code installed then I'd suggest copy the code to a file and opening it in VS Code (then formatting the document).
> If you don't then I can probably fix the indents

Well i tried to fix it but it not working....could u fix it for me. My apologies for cleaning up my indent mess.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@ChunkyProgrammer i tried every shortcut i can find to format document but its not working is there something im missing?

**Comment by ChunkyProgrammer** [comment]
> @ChunkyProgrammer i tried every shortcut i can find to format document but its not working is there something im missing?

I don't believe there's a shortcut for it. It's just a lot easier to spot the mistakes when using vs code or any other code editor (I should have made that more clear). 

I also updated this repo: https://github.com/ChunkyProgrammer/github-issue-form-test with the changes you made (so you can preview what the new issue templates will look like by making issues)

File: .github/ISSUE_TEMPLATE/question.yaml
Comment (PrestonN) [inline .github/ISSUE_TEMPLATE/question.yaml:1]: Same thing here. Should we keep this with the Discussions tab now enabled?

**Review by PrestonN — CHANGES_REQUESTED** [review]
Looking good. As per the discussion in our chat room, it may be worth including the option for the user to include labels for their feature request / bug reports into this PR. This is assuming that these will automatically label the issue once published. If you feel like this should be included in a separate PR, then feel free to do that instead.

Besides that, I've left some small requests for changes. Check out my comments for info.

**Comment by jasonhenriquez** [comment]
As has been discussed on the Matrix dev channel, I suggest that we also add to the bug / feature request templates a section for users to self-label their issues with 1-3 preset qualifiers. 

Here are some recommendations, in no specific order, for the label options on the bug template:
- visual bug
- keyboard control not working
- text/string issue
- content not loading
- accessibility issue
- usability issue
- causes crash
- feature stopped working
- inconsistent behavior
- data loss
- race condition
- only happens in developer mode

**Comment by PikachuEXE** [comment]
Can we get those screenshots updated and get one per screen?
Or are there other ways to preview this?

**Comment by ChunkyProgrammer** [comment]
> Can we get those screenshots updated and get one per screen?
> Or are there other ways to preview this?

You can preview them here:
https://github.com/ChunkyProgrammer/github-issue-form-test/issues/new/choose

but i will add screenshots

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (PrestonN) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:25]: I would also add `.zip`, `.rpm`, `.dmg`, and `other` to this list

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (jasonhenriquez) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:95]: What's the format for linking multiple videos (even if just a delimiter)? Should be stated explicitly.

**Review by jasonhenriquez — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (PikachuEXE) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:93]: I think this should be `textarea` as there might be multiple URLs to videos

File: .github/ISSUE_TEMPLATE/discussion.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/discussion.yaml:1]: If i would ask a question at an announcement it would be related to the announcement. I think we should keep the issues (question/discussions) separate that are not related to the announcement.

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: This is removed therefore i will resolve this.

File: .github/ISSUE_TEMPLATE/discussion.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/discussion.yaml:1]: Alternatively we could also do something like this

Issues are not a place to go ask support questions or start discussions.
Please ask support questions or start discussions on the mailing list or IRC (#freetube:matrix.org) (Web | Matrix)

I also see this allot on other repos

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: This has been removed in the new commit. Therefore i'll mark this as resolved.

File: .github/ISSUE_TEMPLATE/discussion.yaml
Comment (peepopoggers) [reply .github/ISSUE_TEMPLATE/discussion.yaml:1]: Hey, are you going to remove the template?

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/discussion.yaml:1]: Hi, this is still undecided whether we keep or delete them.

**Review by peepopoggers — CHANGES_REQUESTED** [review]
I approve this but I don't think we need templates for discussions or questions. We have the discussions tab for that.

**Comment by peepopoggers** [comment]
I think this PR is really helpful for FreeTube's process of opening issues. If we remove discussions / questions in favor of the dedicated tab, we can keep these templates just in case.

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (peepopoggers) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:10]: You should add - "Make sure you are on the latest release of FreeTube before submitting an issue"

File: .github/ISSUE_TEMPLATE/discussion.yaml
Comment (peepopoggers) [reply .github/ISSUE_TEMPLATE/discussion.yaml:1]: We can keep this template and it can be archived if it is decided that discussions will be removed from issues.

File: .github/ISSUE_TEMPLATE/discussion.yaml
Comment (peepopoggers) [inline .github/ISSUE_TEMPLATE/discussion.yaml:10]: Use capital "I" not "i"

File: .github/ISSUE_TEMPLATE/feature_request.yaml
Comment (peepopoggers) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:33]: I would change this to: Screenshots / Additional Information

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (peepopoggers) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:78]: I would change this to: Screenshots / Additional Information

File: .github/ISSUE_TEMPLATE/question.yaml
Comment (peepopoggers) [reply .github/ISSUE_TEMPLATE/question.yaml:1]: We can keep this template and it can be archived if it is decided that questions will be removed from issues.
I think questions may be better as issues as lots of questions get resolved quickly with one answer, rather than having a discussion about it.

**Review by peepopoggers — CHANGES_REQUESTED** [review]
I have looked down through all of these templates and they all look up to par except the few changes I have requested. When those changes are made, I will be able to approve this PR.

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:78]: Dont think that is a good idea

Check out line 81

https://github.com/FreeTubeApp/FreeTube/commit/aaa336bf48fc6bf20f1172d5f3d6a78aa3cb9bfc

File: .github/ISSUE_TEMPLATE/feature_request.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/feature_request.yaml:33]: To keep it uniform with the bug report template i think that this isnt a good idea.

https://github.com/FreeTubeApp/FreeTube/pull/1593#discussion_r715022409

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:10]: We didnt put it there on purpose. Check out lines 32-36 and line 87. This is enough information for us to know if they are using the latest freetube release

File: .github/ISSUE_TEMPLATE/discussion.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/discussion.yaml:10]: i fail to see where :(

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (jasonhenriquez) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:10]: @efb4f5ff-1298-471a-8973-3d47447115dc This proposed line would be more about reminding the user than providing us new info.

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:10]: ah ok where do u want it above the line or under the line where u need to check the issue tracker

File: .github/ISSUE_TEMPLATE/discussion.yaml
Comment (PrestonN) [reply .github/ISSUE_TEMPLATE/discussion.yaml:1]: I'm still in favor of removing this in favor of the discussions tab.

A quick reminder that any user can create a thread as a discussion, so threads wouldn't have to be limited to the announcement threads that we create. We can also mark discussions as answered which would help others looking for something similar.

If this is the direction that we go, then this template along with the question template would simply be deleted.

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/discussion.yaml:1]: Ah yes didnt even think about it like that. I will remove the templates in next commit!

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:15]: fixed https://github.com/FreeTubeApp/FreeTube/pull/1593/commits/cd65ff0f15f3819f65f37e6d9056dfb96b53619d

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (peepopoggers) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:16]: encountered

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (peepopoggers) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:18]: please change i to I

File: .github/ISSUE_TEMPLATE/feature_request.yaml
Comment (peepopoggers) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:16]: change i to I

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: fixed https://github.com/FreeTubeApp/FreeTube/pull/1593/commits/c0f93e61c55890d329f8f67a3d3cd2bd0581a372

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:18]: i think im really blind. Where exactly?

File: .github/ISSUE_TEMPLATE/feature_request.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/feature_request.yaml:16]: i think im really blind. Where exactly?

Comment (peepopoggers) [reply .github/ISSUE_TEMPLATE/feature_request.yaml:16]: image
Same for other instance

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/feature_request.yaml:16]: hmm i dont understand please explain why this needs to be a capital I

Comment (ChunkyProgrammer) [reply .github/ISSUE_TEMPLATE/feature_request.yaml:16]: @efb4f5ff-1298-471a-8973-3d47447115dc 
The word "I" is always supposed to be a capital letter

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/feature_request.yaml:16]: oh really didnt know about that. will fix!

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/feature_request.yaml]: fixed!

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: Fixed!

**Review by peepopoggers — APPROVED** [review]
Great work :+1:

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> 
> 
> Great work 👍
> After comparing the old format and looking over small details nothing else is needed to merge.

We need to do one more thing and that is figure out the label system not sure if u saw us discussing it in IRC. 
but my mind is completly empty on that so i hope the wizard @ChunkyProgrammer knows of something.

**Comment by ChunkyProgrammer** [comment]
I consulted my magic wizard ball and AFAIK it's not possible. Maybe there's a github workflow that can look at the contents of an issue and auto assign a label based on that? I'm not familiar with GitHub workflows so I wouldn't be much help with that

**Comment by jasonhenriquez** [comment]
> Maybe there's a github workflow that can look at the contents of an issue and auto assign a label based on that? I'm not familiar with GitHub workflows so I wouldn't be much help with that

This would pretty much be the idea. Have the labels be very specific phrases, put them in a required multiple-selection dropdown box, and [use a tool like this](https://github.com/marketplace/actions/issue-auto-labeling-and-assigning) to read it & set the label.

**Comment by ChunkyProgrammer** [comment]
> This would pretty much be the idea. Have the labels be very specific phrases, put them in a required multiple-selection dropdown box, and [use a tool like this](https://github.com/marketplace/actions/issue-auto-labeling-and-assigning) (or GH Actions) to read it & set the label.

Added some labels & the workflow

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Looking good! Just an idea not sure if we want this

but maybe we can also create two labels: releases and nightly

so if the first checkbox is checked the label releases gets added but when then checkbox at the bottom is checked it will overrule the first checkbox and add the label nightly

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@ChunkyProgrammer ready to review?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> As has been discussed on the Matrix dev channel, I suggest that we also add to the bug / feature request templates a section for users to self-label their issues with 1-3 preset qualifiers. 
> 
> Here are some recommendations, in no specific order, for the label options on the bug template:
> - visual bug
> - keyboard control not working
> - text/string issue
> - content not loading
> - accessibility issue
> - usability issue
> - causes crash
> - feature stopped working
> - inconsistent behavior
> - data loss
> - race condition
> - only happens in developer mode

If i understand correctly u also want this on the feature request template?

**Comment by ChunkyProgrammer** [comment]
> @ChunkyProgrammer ready to review?

I think so

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@ChunkyProgrammer i made some comments can u see them?

**Comment by ChunkyProgrammer** [comment]
> @ChunkyProgrammer i made some comments can u see them?

I'm not sure which comment you want me to look at

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (peepopoggers) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:30]: Maybe add another? What if the program gives you a bsod? What if it makes your motherboard melt? What option would you choose for those cases?

In all seriousness though, you should add "other issue" because not everything fits these categories.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
thats weird i made 2 review comments in the bug report file. they are both related to eachother. Thats weird i cant ping u in that comment.

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (ChunkyProgrammer) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:32]: I don't think this drop down should be required

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
i think ur now able to see both the comments

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:30]: im just following instructions :'(
these are not all the labels these also need to be added

https://github.com/FreeTubeApp/FreeTube/pull/1593#issuecomment-921352760

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:32]: im just following instructions :'(
https://github.com/FreeTubeApp/FreeTube/pull/1593#issuecomment-926196910

**Comment by ChunkyProgrammer** [comment]
I can't seem to be able to find your commenta

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Thats weird? Oh i see the comments i made have a pending status. How do i get rid of that :(

**Comment by ChunkyProgrammer** [comment]
Screenshots_2021-09-28-16-58-02

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Omg im so stupid -.-

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Note to @everyone: Dont create labels related to the templates yet. I'd like to have a full list of all the labels for bug and feature request templates first before creating them. This is because we'd need to group labels based on what categorize they belong to otherwise it will look like a mess. Thanks for pointing this out @PikachuEXE.

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: I'll mark this resolved but i didnt added it yet.

https://github.com/FreeTubeApp/FreeTube/pull/1593#issuecomment-930481244

File: .github/ISSUE_TEMPLATE/bug_report.yaml
Comment (peepopoggers) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:115]: should you add a newline?

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:115]: i'm sorry i fail to see why i should. pls explain

Comment (PikachuEXE) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:115]: Ensure new line is present otherwise got some inconsistent diff when different dev edit these files

See `insert_final_newline` on https://editorconfig.org

**Comment by jasonhenriquez** [comment]
My suggestions for feature request labels, in no particular order:
- improvement to existing feature
- new optional setting
- visual improvement
- display more information to user
- ease of use improvement
- support for external software
- new keyboard shortcut

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
i think everything is addressed and it is ready to be merged!

**Comment by PikachuEXE** [comment]
I think we should let @PrestonN have a final review before merge

**Comment by peepopoggers** [comment]
Is there any reason why this should not be merged?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
No u can approve it but the third reviewer spot is reserved for @PrestonN

**Review by PrestonN — APPROVED** [review]
I'm going to go ahead and approve this. There may be some more tweaks for it in the future, but it's leagues better than how everything is setup currently. We can change more to it down the line when needed.

Thanks so much for putting this together!

**Pull request merged**
